app/views/posts/index.html.erb

```diff
@@ -1,9 +1,41 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title>All Posts by User</title>
-</head>
-<body>
-  <h1>Here is a list of posts for a given user</h1>
-</body>
-</html>
\ No newline at end of file
+<h1>Here is a list of posts for a given user</h1>
+<%= link_to user_path(@user), class: "user-row-link" do %>
+  
+    <div class="user-photo">
+      <%= image_tag @user.photo %>
+    </div>
+  <div class="user-row">
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <div class="post-counter">Number of posts: <%= @user.posts.count %></div>
+    </div>
+  </div>
+<% end %>
+
+<% @user.posts do |post| %>
+  <%= link_to user_post_path(@user, post), class: "post-link" do %>
+    <div class="post-row">
+      <h3><%= post.title %></h3>
+      <p><%= truncate(post.text, length: 50) %></p>
+      <div class="post-stats">
+        <span>Comments: <%= post.comments.count %></span>
+        <span>Likes: <%= post.likes.count %></span>
+      </div>
+    </div>
+  <% end %>
+
+ <div class="comments-container">
+  <% @comments.each do |comment| %>
+    <div class="comment">
+      <span class="comment-author"><%= comment.user.name %>:</span>
+      <span class="comment-text"><%= comment.text %></span>
+    </div>
+  <% end %>
+</div>
+
+
+<% end %>
+
+<%= link_to 'Create a new post', new_user_post_path(@user), class: 'see-all-posts-btn' %>
```

Comment:
- [ ] your post index page shows no post even when the user has posts, kindly look at your logic and fix
![Screenshot from 2023-11-03 16-26-33](https://github.com/arnoldnekemiah/blog/assets/66097371/1e9f7017-4f61-4bec-a112-879d0ad86dea)